src/components/interaction-designer/block-editors/BlockOutputBranchingConfig.vue

```diff
@@ -130,17 +91,12 @@
     @Prop({default: 'text-primary'}) readonly labelClass?: string
 
     OutputBranchingType = OutputBranchingType
-    NoValidResponseHandler = NoValidResponseHandler
```

Comment:
The build is failing because `Block.vue` imports this enumeration, but it seems it never uses it.